src/cider/nrepl.clj

```diff
@@ -478,12 +478,32 @@
               :returns {"fn-deps" "A list of function deps."
                         "status" "done"}}}})
 
+(def-wrapper wrap-clojuredocs cider.nrepl.middleware.clojuredocs/handle-clojuredocs
+  {:doc "Middleware to find a documents from ClojureDocs."
+   :handles {"load-clojuredocs-cache!"
```

Comment:
I don't think clients will need to call this explicitly, probably they'll just need the ability to reload the cache.

Reply:
Apart from refreshing cache, I thought it could be used for cache loading in the background (e.g. when connecting nREPL).
But once the cache file is generated, loading of the cache is fast enough, so it seems that just refreshing is good.